munit/shared/src/main/scala/munit/internal/console/Lines.scala

```diff
@@ -16,9 +16,18 @@ class Lines extends Serializable {
   }
   def formatPath(location: Location): String =
     location.path
+
+  def findPath(cwd: String, path: String, max: Int): Path = {
+    val p = Paths.get(cwd).resolve(path)
+    if (Files.exists(p)) p
+    else if (max < 1) sys.error(s"$path was not found")
+    else if (cwd.contains("\\")) findPath(cwd.split("\\").dropRight(1).mkString("\\"), path, max - 1)
```

Comment:
Looks like some of the tests are failing on Windows, would you be able to take a look?

Reply:
Scalafmt and Scalafix are having an issue, too

Reply:
I guess it's failing because the expected value contains / and on Windows it's returning \.